src/SwedbankPay.Sdk/Models/Response/ConsumersResponse.cs

```diff
@@ -1,11 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Client.Models.Response
+﻿namespace SwedbankPay.Sdk.Models.Response
 {
+    using System.Collections.Generic;
+
     public class ConsumersResponse
     {
         public string Token { get; set; }
```

Comment:
Why does a `Response` object have a `public` setter for any of its properties? What does it mean to a consumer of `ConsumerResponse` that `ConsumerResponse.Token` can be changed? Please ensure immutability (preferably with `readonly`, but at least with `private` setters) wherever it's possible to enforce.